CraftTweaker2-API/src/main/java/crafttweaker/CraftTweakerAPI.java

```diff
@@ -271,7 +272,7 @@ public static void registerClass(Class annotatedClass) {
                 try {
                     IBracketHandler bracketHandler = (IBracketHandler) annotatedClass.newInstance();
                     registerBracketHandler(bracketHandler);
-                } catch(InstantiationException | IllegalAccessException ex) {
+                } catch(InstantiationException | IllegalAccessException | IllegalArgumentException | SecurityException ex) {
```

Comment:
Since you reverted the change, these additional catches shouldn't be necessary, should they?